src/algorithm/dynamics.hpp

```diff
@@ -106,7 +106,45 @@ namespace pinocchio
     
     return a;
   }
-  
+
+  ///
+  /// \brief Computes the inverse of the constraint matrix [[M JT], [J 0]].
```

Comment:
You can call this matrix the KKT matrix of the contact dynamics.

Reply:
Done.